dcc-portal-ui/app/scripts/downloader/js/request.js

```diff
@@ -136,8 +136,10 @@
 
         $scope.params.processing = false;
       },function (error) {
-        $scope.params.processing = false;
-        $scope.params.downloadEnabled = false;
+        if(error.status === 503){
+          $scope.params.processing = false;
+          $scope.params.downloadEnabled = false;
+        }
```

Comment:
Not sure I understand this:

> The errorInterceptor function, whenever it returns false, prevents the promise linked to a Restangular request to be executed. All other return values (besides false) are ignored and the promise follows the usual path, eventually reaching the success or error hooks.

Also, not really sure what `responseHandler` is. Is that the local error handler?